hack/release-manifest.py

```diff
@@ -0,0 +1,93 @@
+import os
+import yaml
+import sys
+from pathlib import Path
+
+MANIFEST_DIR = str(sys.argv[1])
+path = os.getcwd()
+release_yaml = path + "/out/release.yaml"
+Path(release_yaml).touch()
+res = list()
```

Comment:
define this is var inside `release_manifest()`, avoid that functions are accessing global vars.

Reply:
We need it as global as it is being used in another function copy_lines_to_file , (needs renaming)

Reply:
Good practice in any language is to avoid having functions with side effects: instead of adding something `rest` withing the called function (and not passing `res` as param), return to the caller the object that will be added to `res` by the caller.